Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM python:3.6
+
+ARG NEW_USER_UID
+ARG NEW_USER
+
+RUN apt-get -y update && \
+    apt-get install -y vim openssh-client python-tox && \
```

Comment:
out of curiosity: Why do we need `vim`?

Reply:
This was just in case something goes weird inside the container and we need to edit a file, which `exec`'ed inside the container. Since we are binding everything locally this is generally not needed, but I though maybe there's this edge case where we need to quickly editing something (had to do it myself :) ).

Reply:
I see. Makes sense.